### ISSUE TITLE
Move CPC meetings to bi-weekly cadence

### ISSUE THREAD

**Issue by joesepi** [openjs-foundation/cross-project-council#680]
Our agendas have become lighter where it seems a weekly meeting is now overkill. I've been waiting for this day for over 2 years. Yay! 🎉 

My suggestion is that we alternate bi-weekly times as we are doing now between 12pm and 2pm ET. (sorry for US East centric framing.) And in addition to that, we use the off weeks to have working sessions or slot in any needed ad-hoc meetings.

It would look like this:
week 1, 17:00 UTC: CPC Meeting
week 2, 17:00 UTC: CPC Working Meeting
week 3, 19:00 UTC: CPC Meeting
week 4, 19:00 UTC: CPC Working Meeting
and so on...

Note: we would need to confer with @openjs-foundation/standards group to coordinate new meeting schedule and not step on their schedule.

**Comment by joesepi** [comment]
During today's meeting, I proposed that we start this new cadence in two weeks -- December 1. This will give us more time to circulate the news and to make sure we coordinate with @openjs-foundation/standards friends.

**Comment by joesepi** [comment]
We will start this in January 2021.

**Comment by joesepi** [comment]
Just to wrap this up:

- I have updated the calendar to have the CPC meeting times be bi-weekly as stated above.
- No changes have been made with regards to the Standards team bi-weekly meeting
- After a conversation with Jory, we decided to not put the working sessions on the calendar. The thinking is that there shouldn't be a meeting if there is no agenda. So the intent is to discuss the next working session at each CPC meeting and flesh out an agenda and create the meeting at that time. This can be tweaked as we move forward and see how the process plays out.
